### PR TITLE
Update Code - DocBlock Standard

### PR DESCRIPTION
**Pull request by edenduong**
## Purpose of this pull request

This pull request (PR) will remove the Old Magento 1 code in the  DocBlock Standard Page.
Update to Magento 2 code.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.3/app/code/Magento/Backend/Model/Auth.php#L141

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by edenduong** [comment]
@jeff-matthews : I have just merge into 1 PR. You can check it. Thanks!

**Comment by jeff-matthews** [comment]
Thanks @edenduong! Coding Standards require review from Magento architects, so I'm adding @buskamuza.

**Comment by dobooth** [comment]
Hi @edenduong Thanks for the contribution. We recently stopped taking contributions for v2.2, as it is entering EOL. Please update the changes to the v2.3 file. Also, since it is dealing with coding standards, we are asking for architects to review. Thanks.

File: src/guides/v2.2/coding-standards/docblock-standard-general.md [hunk 1]
```diff
@@ -444,31 +444,28 @@ For example:
...
+    if ($this->_coreFileStorageDb->checkDbUsage()) {
+        $this->_directoryDatabaseFactory->create()->deleteDirectory($path);
     }
-
-    $io = new Varien_Io_File();
-
-    if (Mage::helper('Mage_Core_Helper_File_Storage_Database')->checkDbUsage()) {
-        Mage::getModel('Mage_Core_Model_File_Storage_Directory_Database')->deleteDirectory($path);
+    if (!$this->isPathAllowed($path, $this->getConditionsForExcludeDirs())) {
+        throw new \Magento\Framework\Exception\LocalizedException(
```
Comment (buskamuza) [inline src/guides/v2.2/coding-standards/docblock-standard-general.md]: This is explicit throwing of an exception. The section is about implicit throwing. An example could be if there would be a method like `$this->throwException()` or something like this.

**Comment by dobooth** [comment]
Hi @edenduong Any update on this PR?

**Comment by dobooth** [comment]
@edenduong Still out there? Any updates?

**Comment by edenduong** [comment]
@dobooth: Sorry for late response. I have just change the docs. 
Thanks!

File: src/guides/v2.2/coding-standards/docblock-standard-general.md (hunk 1 above)
Comment (edenduong) [reply src/guides/v2.2/coding-standards/docblock-standard-general.md]: I have change the example. Please check it. Thanks!

**Comment by dobooth** [comment]
running tests

**Pull request merged**